cdap-integration-test/src/main/java/co/cask/cdap/test/remote/RemoteApplicationManager.java

```diff
@@ -127,25 +77,7 @@ public MapReduceManager startMapReduce(final String programName) {
 
   @Override
   public MapReduceManager startMapReduce(final String programName, Map<String, String> arguments) {
```

Comment:
Oh oops, just saw that the call is embedded inside the ``DefaultMapReduceManager`` constructor. Maybe pull it out for consistency?